sof-defconfig

```diff
@@ -5,6 +5,7 @@ CONFIG_SND_SOC_SOF_TOPLEVEL=y
 CONFIG_SND_SOC_SOF_PCI=m
 CONFIG_SND_SOC_SOF_ACPI=m
 CONFIG_SND_SOC_SOF_OF=m
+CONFIG_SND_SOC_SOF_CLIENT=y
```

Comment:
is this y for boolean or can it be m? That will impact how the code is compiled.

Reply:
Its a boolean. so it cant be m, right?